### PR TITLE
fix up the ci-cd assistant docs

### PR DESCRIPTION
**Pull request by susanev**
we received some feedback on the ci/cd docs being confusing and not clearly describing where to find the assistant.
this cleans them up a bit and adds a screenshot that reveals the context in which it exists.

### CODE REVIEW

File: themes/default/content/docs/intro/pulumi-service/ci-cd-integration-assistant.md [hunk 1]
```diff
@@ -17,46 +17,31 @@ to [Organizations]({{< relref "/docs/intro/pulumi-service/organizations" >}}), n
 
 <!--more-->
 
-A version control system (VCS) provides team collaboration capabilities and ensures that the source code for your Pulumi project is not
-just on a single developer's machine. Using a CI/CD system makes your team more productive, by automatically deploying your Pulumi stacks.
-So that code is delivered to production -- meaning less of a lag between commits and what your end users are seeing.
-
-The assistant contains a guided experience to help teams configure a VCS as well as automate it with a
+A version control system (VCS) provides team collaboration capabilities and ensures that the source code for your Pulumi project is not on a single developer's machine. Using a CI/CD system makes your team more productive, by automatically deploying your Pulumi stacks. The assistant contains a guided experience to help teams configure a VCS as well as automate it with a
```
Comment (stevesloka) [inline themes/default/content/docs/intro/pulumi-service/ci-cd-integration-assistant.md:20]: nit: A helpful thing with markdown is to keep a sentence on its own line, that way for reviews, you get the diff of just that line and not a huge change.

File: themes/default/content/docs/intro/pulumi-service/ci-cd-integration-assistant.md [hunk 2]
```diff
@@ -17,46 +17,31 @@ to [Organizations]({{< relref "/docs/intro/pulumi-service/organizations" >}}), n
...
 CI/CD pipeline.
 
 The assistant has two options to help you to get started with integrations:
 
 * Use a single service that provides both a version control system and CI/CD pipelines
-* Or, just get the starter workflow for a CI/CD service of your choice customized for your stack
+* Or, get the starter workflow for a CI/CD service of your choice customized for your stack
```
Comment (stevesloka) [inline themes/default/content/docs/intro/pulumi-service/ci-cd-integration-assistant.md:26]: Why would I pick the first vs the second? Not sure if those docs have already explained this. =)

**Review by stevesloka — APPROVED** [review]
/lgtm

File: themes/default/content/docs/intro/pulumi-service/ci-cd-integration-assistant.md (hunk 2 above)
Comment (susanev) [reply themes/default/content/docs/intro/pulumi-service/ci-cd-integration-assistant.md:26]: 🤷🏻 can try and fix

Comment (susanev) [reply themes/default/content/docs/intro/pulumi-service/ci-cd-integration-assistant.md]: okay what that meant was that you can use the same service for both version control and CI/CD or you can use different services for those. and the assistant supports both. ive tried to make this more clear.

Comment (susanev) [reply themes/default/content/docs/intro/pulumi-service/ci-cd-integration-assistant.md]: why you would do that i think depends on your org needs and things

**Pull request merged**